Documentation/operations/upgrade.rst

```diff
@@ -320,6 +320,9 @@ communicating via the proxy must reconnect to re-establish connections.
 * CiliumIdentity CRD does not contain Security Labels in metadata anymore except for the namespace label.
 * The support for Envoy Go Extensions (proxylib) is deprecated, and will be removed in a future release.
 * The kube_proxy_healthz endpoint no longer requires Kubernetes control plane connectivity to succeed.
+* A new opt-out option ``policy-default-local-cluster`` has been added in Cilium 1.18 and will become a default option in Cilium 1.19.
+  If you are using Cilium ClusterMesh and network policies enabling this option would certainly result in a breaking change for your existing policies!
+  See :ref:`policies_clustermesh` for details and migrations recommendations.
```

Comment:
I don't see the migration recommendation, am I missing something?

Reply:
It's sort of implicitly said there but yes will do a dedicated section like you suggested in your other comment!